services/horus.coffee

```diff
@@ -0,0 +1,64 @@
+fs = require 'fs'
+log = require 'loglevel'
+_ = require 'lodash'
+
+config = require '../config'
+
+SEPARATOR = '.'
+CONVERSIONS_FILE = "#{config.LOG_DIR}/flak_cannon_conversions_horus.tsv"
+
+# NOTE: These cannnot be changed. Create a new log file instead
+CONVERSION_COLUMNS = [
+  { key: 'event', name: 'event' }
+  { key: 'userId', name: 'user_id' }
+  { key: 'params', name: 'params+' }
+  { key: 'uniq', name: 'uniq' }
+  { key: 'ip', name: 'ip' }
+  { key: 'namespace', name: 'namespace' }
+  { key: 'meta', name: 'meta+' }
+]
+
+getColumns = ->
+  _.pluck CONVERSION_COLUMNS, 'name'
+  .join '\t'
+
+flattenObject = (obj, prefix = '') ->
+  _.reduce obj, (result, val, key) ->
+    if _.isPlainObject val
+      result.concat flattenObject val, prefix + key + SEPARATOR
+    else
+      result.concat prefix + key + '=' + val
```

Comment:
isn't it possible that val has a space in it? which would break L40, L42 where it's separated by space (it wouldn't break those lines, it'd break our data)